### ISSUE TITLE
Language selection doesn't always reload the image

### ISSUE THREAD

**Issue by dstanko001** [pwc-technology-be/eu-captcha#77]
Not all the languages are reloading the captcha when you select a new language from the drop-down menu. I will open a new issue for this

**Comment by dstanko001** [comment]
I am still getting some CORS issues with some of the languages 
Access to XMLHttpRequest at 'https://6rwxgmwvl6.execute-api.eu-west-1.amazonaws.com/api/captchaImg/?lang=cs' from origin 'http://ec2-34-251-252-103.eu-west-1.compute.amazonaws.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

**Comment by Eddy-Hoevenaers** [comment]
I changed the implementation to use the unicode, before we used typed characters. The CORS issue is also solved

**Comment by Eddy-Hoevenaers** [comment]
This issue is fixed, all languages load on the getCaptcha call
